.github/workflows/build-images.yml

```diff
@@ -360,12 +360,11 @@ jobs:
         # pushing it up to an external repository.
         if [ $GITHUB_EVENT_NAME != 'pull_request' ]
         then
-          docker manifest inspect ${OPERATOR_IMG} > /dev/null 2>&1
-          EXIT_CODE=$?
+          EXIT_CODE="$(docker manifest inspect ${OPERATOR_IMG} > /dev/null ; echo $?)"
```

Comment:
Why do we remove "2>&1"?

Reply:
Mainly for debugging purposes, if we have 2>&1, then the error.output will not be printed on the log. We simply want to dump the success standard output from. 
With existing setup: > /dev/null
If image exist, then its success, standard.output will be redirect to  /dev/null, no output is ok.
If image doesn't exist, we will still see logs in error.output. Such as "manifest unknown", or other types like unauth, etc. So we know the true reason why the command returns the return code 1.

With the old setup: > /dev/null 2>&1
All output (standard.output, error.output) will be dumped, its hard for debugging.

Both setup works, because we just need the return code from last command. But use only "/dev/null" provides more useful info to us.